template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/oozie/OozieHAConfigProviderTest.java

```diff
@@ -25,7 +25,7 @@ public class OozieHAConfigProviderTest {
     public void testGetServiceConfigsWithSingleRolesPerHostGroup() {
         String inputJson = getBlueprintText("input/clouderamanager-db-config.bp");
         CmTemplateProcessor cmTemplateProcessor = new CmTemplateProcessor(inputJson);
-        TemplatePreparationObject preparationObject = getTemplatePreparationObject(inputJson, cmTemplateProcessor, 1);
+        TemplatePreparationObject preparationObject = getTemplatePreparationObject(inputJson, cmTemplateProcessor, 1, false);
```

Comment:
I cannot find any testcase where SSL is enabled, and also the JDBC URL property is not checked anywhere.